src/content.ts

```diff
@@ -188,4 +188,9 @@ export const LangPoweredBy: PoweredBy[] = [
     projectUrl: 'https://distrobox.privatedns.org/',
     altText: 'Distrobox',
   },
+  {
+    imageUrl: 'https://raw.githubusercontent.com/containers/common/main/logos/bootc-logo-full-vert.png',
+    projectUrl: 'https://containers.github.io/bootc/',
+    altText: 'bootc',
```

Comment:
Download this logo and put it in the same place as the others so it's served from a relative path from this repo instead of remotely.